README.md

```diff
@@ -7,40 +7,40 @@ This service enables secure token exchange between GitHub Actions OIDC tokens an
 - Verifies GitHub Actions OIDC tokens
 - Creates Snyk service accounts if they don't exist
 - Generates temporary Snyk tokens with 10-minute TTL
-- Secure token exchange with proper validationgit co
-
-# Example GitHub Action Using Service
```

Comment:
I'd prefer to lead with a simple example of using this using github to immediately show the purpose of the script. This could be the "Minimal example" and your example at the end a more complete example